molpipeline/estimators/chemprop/models.py

```diff
@@ -262,10 +277,11 @@ def __init__(
             Parameters set using `set_params`.
             Can be used to modify components of the model.
         """
+        bmp_args, ffn_args, kwargs = self._split_args(**kwargs)
         if model is None:
-            bond_encoder = BondMessagePassing()
+            bond_encoder = BondMessagePassing(**bmp_args)
```

Comment:
@JenniferHem I think this is not necessary, as `ABCChemprop` calls `self.set_params(**kwargs)` which should recursively set the parameters to the model components.